kotlin-oauth-authenticator-archetype/src/main/resources/archetype-resources/src/main/kotlin/authentication/RedirectUriUtil.kt

```diff
@@ -0,0 +1,33 @@
+package ${package}.authentication
```

Comment:
missing copyright

Reply:
None of the files in our archetypes have copyright notices, that's why I haven't added one here.